chain/bitcoind_zqm_events.go

```diff
@@ -319,3 +402,189 @@ func (b *bitcoindZMQEvents) txEventHandler() {
 		}
 	}
 }
+
+// sequenceEventHandler reads sequnce events from the ZMQ block socket and
+// forwards them along to the current rescan clients.
+//
+// NOTE: This must be run as a goroutine.
+func (b *bitcoindZMQEvents) sequenceEventHandler() {
+	defer b.wg.Done()
+
+	log.Info("Started listening for bitcoind sequence notifications "+
+		"via ZMQ on", b.sequenceConn.RemoteAddr())
+
+	// Set up the buffers we expect our messages to consume. For sequence
+	// events, the body is structured as the following based on the type of
+	// message:
+	// <32-byte hash>C :                 Blockhash connected
+	// <32-byte hash>D :                 Blockhash disconnected
+	// <32-byte hash>R<8-byte LE uint> : Transactionhash removed from
+	// 	mempool for non-block inclusion reason
+	// <32-byte hash>A<8-byte LE uint> : Transactionhash added mempool
+	//
+	// We'll allocate a fixed data slice that we'll reuse when reading
+	// transactions from bitcoind through ZMQ. There's no need to recycle
+	// this slice (zero out) after using it, as further reads will
+	// overwrite the slice and we'll only be deserializing the bytes
+	// needed.
+	var (
+		eventHash [32]byte
+		eventType [1]byte
+	)
+
+	for {
+		// Before attempting to read from the ZMQ socket, we'll make
+		// sure to check if we've been requested to shut down.
+		select {
+		case <-b.quit:
+			return
+		default:
+		}
+
+		// Poll an event from the ZMQ socket.
+		var (
+			bufs = [][]byte{eventHash[:], eventType[:]}
```

Comment:
Why does this omit the command?

Reply:
This should be `bufs = [][]byte{command[:], data, seqNum[:]}`

Then we'd go ahead and parse `eventHash`, `eventType`, and `eventSeq` (not `seqNum`) from `data`